src/app/parcels/ActionBar/ActionAndStatusBar.test.tsx

```diff
@@ -0,0 +1,169 @@
+import React from "react";
+import { render, screen, fireEvent, waitFor } from "@testing-library/react";
+import { expect, it } from "@jest/globals";
+import "@testing-library/jest-dom/jest-globals";
+import ActionAndStatusBar, {
+    ActionAndStatusBarProps,
+} from "@/app/parcels/ActionBar/ActionAndStatusBar";
+import { ParcelsTableRow } from "../parcelsTable/types";
```

Comment:
I think absolute route importing is preferred i.e @/app/parcels/parcelsTable instead of ../parcelsTable